src/components/display/fewlines/ErrorFallback/ErrorFallbackComponent.tsx

```diff
@@ -45,10 +47,13 @@ const Wrapper = styled.div<{ statusCode: number }>`
   padding: ${({ theme }) => theme.spaces.xs};
   margin-top: ${({ statusCode, theme }) =>
     statusCode !== 404 ? theme.spaces.s : ""};
+
+  h1 {
+    margin-bottom: 3rem;
+    padding: 0;
+  }
```

Comment:
😢 

Shouldn't that kind of things be global?

Reply:
Well, the global `h1` is for all h1 in the app. This one is the only particular case. Do you want me to make them both `p`? I used an `h1` because this component is the only one rendered in the page, and I wanted an `h1`.

Reply:
I would ask the other way around, why is that one `h1` different from the others?

Reply:
Because in the app, we use a compact style. On the 404 page, we have huge empty space, so I thought making them breathe a little wouldn't hurt.

I can use a flex box to give the 2 lines some room if you prefer.

Reply:
41f19b5